react/forms/step2.md

```diff
@@ -0,0 +1,122 @@
+In PatternFly 4, demos are used to show how Patternfly’s components and layouts can be put together to build more complex structures.
+
+They're made entirely from components and layouts.
+
+In this exercise, we'll be using a form demo to create a new form for a web application.
+
+The image below is what we'll end up with when we are done:
+
+<img src="forms/assets/step3.png" alt="Completed form with title field and email validation" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
```

Comment:
I would prefer to see the resulting image in step 1. Leave step two to the task at hand.

See https://katacoda.com/patternfly/courses/charts/area-chart

Reply:
This is the pattern Dana and I used when writing the React tutorials. We did user testing on them and the users preferred seeing what they would be building.

Reply:
I'm only speaking about showing the "final" image in step 1. Step 2 already has an image at the bottom showing what the user is building for that step.

Reply:
This is what we came up with for step 1 of charts.

![Screen Shot 2019-10-10 at 1 11 30 PM](https://user-images.githubusercontent.com/17481322/66590868-b570cd80-eb5f-11e9-8d69-b7a5d544575b.png)

Reply:
Spoke to Dana about moving the "final" image to step 1 (the introduction) and he is in agreement